playwright/src/test/java/com/microsoft/playwright/TestAutoClose.java

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.playwright;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * This test simply makes sure that all 4 main interfaces implement AutoCloseable. If it compiles, then it passes.
+ */
+public class TestAutoClose {
+  @Test
+  void shouldAllowUsingTryWithResources() throws Exception {
+    try (Playwright playwright = Playwright.create();
+         Browser browser = playwright.chromium().launch();
```

Comment:
This will only run it for chromium, can you update the test similar to [`TestBrowser`](https://github.com/microsoft/playwright-java/blob/master/playwright/src/test/java/com/microsoft/playwright/TestBrowser.java) so that it launches the browser which is configure via `BROWSER` env?